src/startupscriptgenerator/src/restorePackages.sh

```diff
@@ -1,32 +0,0 @@
-#!/bin/bash
```

Comment:
I'm just curious about this file. Do we just not need this anymore because of your change, or is it a separate clean up we're doing?

Reply:
This restorePackages.sh is installing "dep" tool and running dep command. We'll no longer need this once we deprecated "dep" and move to "go mod".

Reply:
Thanks for clarifying!